lib/iex/lib/iex.ex

```diff
@@ -248,7 +248,7 @@ defmodule IEx do
   Now we are connected into the remote node, as the prompt shows us,
   and we can access the information and modules defined over there:
 
-      rem(foo@macbook)1> Hello.world
+      iex(foo@HOST)1> Hello.world
       "it works"
```

Comment:
This should be `"it works!"` (with exclamation mark)
```suggestion
       "it works!"
```

Reply:
Thank you for pointing this out, @fertapric 🙇